libs/model/src/models/contest_manager.ts

```diff
@@ -19,17 +19,28 @@ export default (sequelize: Sequelize.Sequelize) =>
         type: Sequelize.STRING,
         allowNull: false,
       },
+      name: {
+        type: Sequelize.STRING,
+        allowNull: false,
+      },
+      image_url: {
+        type: Sequelize.STRING,
+        allowNull: false,
+      },
+      funding_token_address: { type: Sequelize.STRING },
+      prize_percentage: { type: Sequelize.FLOAT },
```

Comment:
Migration is showing that `prize_percentage` and `payout_structure` are of type `DOUBLE PRECISION`. The equivalent in Sequelize seems to be `Sequelize.DOUBLE`. `Sequelize.FLOAT` would be `REAL`. Are the Sequelize docs misleading (seems like this would be caught in CI if it was wrong)?

Reply:
yep, seems like sequelize defaults FLOAT to float8=double in PG... changing to DOUBLE just in case, but any discrepancy would've been captured by model reconciliation tests.